kratos/python_scripts/multistage_orchestrators/sequential_multistage_orchestrator.py

```diff
@@ -0,0 +1,175 @@
+                    err_msg = "'load_from_checkpoint' is expected to be null or a string with the loading point."
+                    raise TypeError(err_msg)
+                else:
+                    loading_point = self.GetProject().GetSettings()["orchestrator"]["settings"]["load_from_checkpoint"].GetString()
+                    self.GetProject().Load(loading_point)
+
+        # Run the stages list
+        for stage_name in self.__GetExecutionList():
+            # Check current stage settings
+            self.CheckStageSettings(stage_name)
+
+            # Set up and check current stage instance
+            current_stage = self.CreateStage(stage_name)
+
+            # Execute current stage preprocess
+            self.RunCurrentStagePreprocess(stage_name)
+
+            # Run (solve) current stage
+            current_stage.Run()
+
+            # Get the final data dictionary
+            self.GetProject().GetOutputData()[stage_name] = current_stage.GetFinalData()
+
+            # Execute current stage postprocess
+            self.RunCurrentStagePostprocess(stage_name)
+
+            # Delete current stage instance
+            del current_stage
+
+            # Output validated simulation settings (with defaults) as simulation report
+            output_settings_name = self.__PrepareOutputSettings(stage_name)
+
+            # Check the current stage is to be checkpointed and save it if so
+            if stage_name in self.__GetStagesToCheckpointList():
+                save_folder_name = self.__GetStagesCheckpointFolder()
+                self.GetProject().Save(save_folder_name, stage_name, output_settings_name)
+
+
+    def GetNumberOfStages(self):
+        '''Returns the number of stages.'''
+
+        return len(self.__GetExecutionList())
+
+    def __GetExecutionList(self):
+        '''Creates and returns the execution list.
+        This method creates the execution list, either from a user-defined execution list or,
+        if this is not provided, from the stages declaration order.
+        '''
+
+        # Check if the execution list has been already created
+        if not hasattr(self, "_execution_list"):
+            # Default case in which the execution list is provided by the user
+            if self.GetProject().GetSettings()["orchestrator"]["settings"].Has("execution_list"):
+                self._execution_list = self.GetProject().GetSettings()["orchestrator"]["settings"]["execution_list"].GetStringArray()
+            # If not provided, create an auxiliary execution list from the stages declaration order
+            else:
+                KratosMultiphysics.Logger.PrintInfo("'execution_list' is not provided. Stages will be executed according to their declaration order.")
+                self._execution_list = list(self.GetProject().GetSettings()["stages"].keys())
+
+        return self._execution_list
+
+    def __GetStagesToCheckpointList(self):
+        '''Creates and returns the stages checkpoint list.
```

Comment:
If it creates then I would not call it Get...

Reply:
(same comment)

Reply:
So I would state it in the documenation

Reply:
Done.